templates/workflow/level1_list.html

```diff
@@ -0,0 +1,145 @@
+<div id="level_1_list">
+  <!-- Sub navigation -->
+  <div class="sub-navigation">
+    <div class="sub-navigation-header">
+      <h4 class="page-title">[[ level_1_label ]] List</h4>
+    </div>
+    <div class="sub-navigation-actions">
+      <div class="sub-nav-item">
+        <div class="btn-group" role="group" aria-label="...">
+          <a
+            role="button"
+            class="btn btn-sm btn-primary"
+            id="show-modal"
+            @click="toggleModal()"
+          >
+            <i class="fa fa-plus"></i> [[ level_1_label ]]
+          </a>
+        </div>
+        <!-- Program create modal -->
+        <modal v-if="showModal" @close="showModal = false">
+          <div slot="body">
+            {% include './modals/add_level1_modal.html' %}
+          </div>
+          <div slot="footer">
+            {% include 'ui-components/quick_entry_modal_footer.html' %}
+          </div>
+          <h4 slot="header">[[modalHeader]]</h4>
+        </modal>
+        <!-- Program delete modal -->
+        <modal v-if="showDeleteModal" @close="showDeleteModal = false">
+          <h4 slot="header">[[modalHeader]]</h4>
+          <div slot="body">
+            <p>
+              Are you sure you want to delete <b>[[ itemToDelete.name ]]</b>
```

Comment:
Please add question mark here

Reply:
ACT-658 calls for implementing the programs create modal using Vue.

The sectors field is a multiple select field, inclusion of this property **multiple=true**. If I remove this, it will be a single select field.

That is the Vue datepicker component I could find that came highly recommended. I don't know whether in activity we have implemented a date field using vue for me to borrow from

Reply:
@IvyMMutiso It strange to have different styling for the same feature. If you go to staging you can see the how the multi-select dropdown is setup currently. Can you please try to mirror that? We have multi-select on many other pages, like program fund code, projects edit page has site locations, stakeholders and many others. So it will be confusing to have it differently. Let me know what you think.

Same for date picker, we have it in so many places and already we have 2 different styling. It will be more messy to add a third kind of styling! I suggest try to mirror what is already there on the add program modal (you can see the example in staging). If that's not possible please try to mirror the other data picker that is in the Vue component like Indicators > add results

Reply:
Date field refactored